src/server/search/doc_accessors.h

```diff
@@ -37,6 +38,33 @@ struct BaseAccessor : public search::DocumentAccessor {
   indexed field
   */
   virtual SearchDocData SerializeDocument(const search::Schema& schema) const;
+
+  virtual AccessResult<VectorInfo> GetVector(std::string_view active_field) const {
```

Comment:
please move the implementation to cc file

Reply:
Oh, yes, and this